.env.backup

```
@@ -0,0 +1,16 @@
+PORT=3000
+
+# Database
+# For local development with Docker:
+DATABASE_URL=postgresql://postgres:postgres@localhost:5432/kime_db?schema=public
+
+# For Supabase (replace with your actual Supabase connection string):
+# DATABASE_URL=postgresql://postgres:[YOUR-PASSWORD]@[YOUR-PROJECT-REF].supabase.co:5432/postgres?schema=public
+
+# Docker PostgreSQL (optional, only if using docker-compose)
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)